package.json

```diff
@@ -33,7 +34,7 @@
     "prettier:cli": "node packages/build/bin/run-prettier \"**/*.ts\"",
     "prettier:check": "npm run prettier:cli -- -l",
     "prettier:fix": "npm run prettier:cli -- --write",
-    "clean": "lerna run clean --loglevel=silent --parallel",
+    "clean": "lerna run clean --loglevel=silent --parallel && node bin/prune-non-packages",
```

Comment:
Is the exclusion of lerna package folders important enough to justify the cost of maintaining the new prune script? IMO, all our packages are removing `dist` folder as part of `npm run clean` anyways, therefore there is little harm done if our top-level "clean" script tries to remove them again.

I am proposing to simplify this command to something like following (please replace `rimraf` with the tool we have in `@loopback/build`, I don't remember which one it is):

```
"clean": "lerna run clean --loglevel=silent --parallel && rimraf \"packages/*/dist\""
```

Thoughts?